src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -58,6 +58,9 @@
 #include "avtBlueprintTreeCache.h"
 #include "avtBlueprintDataAdaptor.h"
 
+#ifdef _WIN32
```

Comment:
Hmm...sorry about that. You know, I didn't even think to go look at windows compatability here. I just now did that and got a bunch of hits...

```
% find . -name '*.[Ch]' -exec grep strcasecmp {} \; -print
#define strcasecmp stricmp
                    if (strcasecmp(line, "true") == 0)
                    divideBrick = (strcasecmp(line, "true") == 0);
./databases/BOV/avtBOVFileFormat.C
// for strcasecmp
./databases/ANSYS/avtANSYSFileFormat.C
     strcasecmp(getenv("CARPETHDF5_CACHE_METADATA"), "no") == 0)
./databases/CarpetHDF5/avtCarpetHDF5FileFormat.C
#  define STREQUAL(a,b)              strcasecmp(a,b)
./databases/Miranda/avtMirandaFileFormat.C
#define strcasecmp stricmp
            if (strcasecmp(ex_var_name, list[i+q]))                    \
./databases/Exodus/avtExodusFileFormat.C
  #define strcasecmp stricmp
            if (!strcasecmp(expr_substrs[1].c_str(),"curve"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"scalar"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"vector"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"tensor"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"array"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"material"))
            else if (!strcasecmp(expr_substrs[1].c_str(),"species"))
./databases/VTK/avtVTKFileReader.C
#  define STREQUAL(a,b)              strcasecmp(a,b)
./databases/Nek5000/avtNek5000FileFormat.C
#define strcasecmp stricmp
    if (!strcasecmp(mode, "FORMATTED")) return ASCII;
    if (!strcasecmp(mode, "ASCII")) return ASCII;
    if (!strcasecmp(mode, "BINARY")) return BINARY;
    if (!strcasecmp(mode, "UNFORMATTED")) return BINARY;
./databases/DDCMD/object.C
            strcasecmp(n_expr["consumer"].as_string().c_str(), "visit"))
./databases/Blueprint/avtBlueprintFileFormat.C
#define stricmp strcasecmp
./tools/third_party/qtssh/unix/unix.h
#define strcasecmp stricmp
            if (strcasecmp(sflag, "float") == 0)
            else if (strcasecmp(sflag, "native") == 0)
            else if (strcasecmp(sflag, "double") == 0)
//   strcasecmp does not exist on Windows so I made it use _strnicmp.
./visitpy/common/visitmodule.C
        if (!strcasecmp(colorName, namedColors[i].n))
./common/state/ColorAttribute.C
                (strcasecmp(idsOrNames[0].c_str(), "None") == 0);
./sim/V2/runtime/SimEngine.C
```

I wonder...does it make sense to fix all cases with function in `StringHelpers.[Ch]`.